.github/workflows/ci.yaml

```diff
@@ -38,6 +38,33 @@ jobs:
           fi
           exit $RESULT
 
+  mac-rendering:
```

Comment:
This seems to take minutes 1-3 to run, while the other steps are under 1 min.
i'm fine to remove this and see if the generate examples.sh needs more changes.